src/examples/header-has.html

```diff
@@ -0,0 +1,32 @@
+<html>
+  <head>
+      <script src="https://learn.staging.concord.org/assets/application.js"></script>
+      <link rel="stylesheet" href="https://learn.staging.concord.org/assets/learn/stylesheets/application.css">
+      <script src="./portal-api.js"></script>
+      <style>
+        .landing-container .home-page-content { width: 100% }
+      </style>
+  </head>
+  <body>
+    <div id="test-header" style="text-align: left;"></div>
+
+    <script>
+      Portal.currentUser = {}
+      Portal.currentUser.isLoggedIn = true
+      window.onload = function () {
+        PortalPages.renderPageHeader({
+          oauthProviders: [
+              { name:           "google",
+                display_name:   "Google",
+                directPath:     "http://google.com" },
+              { name:           "foo",
+                display_name:   "Foo",
+                directPath:     "http://foo.com" }
+            ],
+          theme: 'has'
```

Comment:
Looks like the only difference between header.html and header-has.html is the "theme" is set to 'has'. Too bad there isn't an obvious way to codify all but the theme and have these use some sort of shared "base-header.html", but I don't see an obvious way to do that.

Reply:
Yeah I can't think of something that will make this better to read. The config could be moved into another file like portal-api.js and then modified in each of these html pages. But I think that make it more confusing when looking for an example of a header.

I do think these examples can use some more work at some point so the dependency on react is improved and the dependency on the portal/application.js that some have is removed. Also it might be possible to remove the html files altogether and just use .js files for the examples. But I wanted to get more examples setup to see what would make the most sense.